ocrd_validators/ocrd_validators/workspace_validator.py

```diff
@@ -171,7 +186,7 @@ def _validate_imagefilename(self):
         self.log.debug('_validate_imagefilename')
         for f in self.mets.find_files(mimetype=MIMETYPE_PAGE):
             if not is_local_filename(f.url) and not self.download:
-                self.report.add_notice("Won't download remote PAGE XML <%s>" % f.url)
+                self.log.warning("Won't download remote PAGE XML '%s'", f.url)
```

Comment:
Yes, that is probably better logged than written to the validation report.

Reply:
Like I already said in https://github.com/OCR-D/core/issues/450!